### ISSUE TITLE
Create SVG from remote

### ISSUE THREAD

**Issue by alvesl** [GeraudBourdin/Ti.SvgView#9]
Hello!
I've been trying to manually download a SVG to local storage and open it but I always get

```
[ERROR] :  The application has crashed with an uncaught exception 'NSInternalInconsistencyException'.
[ERROR] :  Reason:
[ERROR] :  Invalid parameter not satisfying: newSource != nil
```

I assume the module is unable to find the file. I've tried loading it (after writing to local) with `nativePath`, and also with `resolve()`.
Can you point me on a direction here? I saw that the original SVGKit module allows creation from both remote and also from memory, this could be a way out too.

Thanks for all the help!

**Comment by GeraudBourdin** [comment]
Please see the demo app at  : https://github.com/GeraudBourdin/Ti.SvgView-TestApp
There is a complete sample on how to use it.
Svg files needs to be on the /app/assets directory.

**Comment by alvesl** [comment]
Hello!
Thanks for the answer. Yes, I've been able to use it correctly if I place my SVGs in the ```/assets``` directory.
Although that works, I need to be able to load SVGs dynamically, from a remote url, as an example, I would want to be able to download and load this SVG: https://upload.wikimedia.org/wikipedia/commons/d/d8/Compass_card_(de).svg

I saw that the SVGKit (for iOS) has the ability to load SVG's from a remote website, or even from memory (after downloading the SVG). If none of those optios are possible, why is it that I cannot load an asset from the `appData` directory?
Thanks

**Comment by falkolab** [comment]
@GeraudBourdin can you implement ability to load from `string`, `blob` source for both platforms? It will be sumptuously =)

**Comment by DaKaZ** [comment]
+1 - being able to load a remote resource would fantastic!  What can we do to help?

**Comment by GeraudBourdin** [comment]
@alvesl : yes SVGKit can do that, but not in the android version. That's why i didn't included it, as you can do it with JS code. I'll have a look at it as soon as i have enough time :)
@DaKaZ : you could send pull request ! :)

**Comment by focussing** [comment]
Dear all,
Any successful attempt in loading a remote SVG file?

**Comment by falkolab** [comment]
@focussing https://github.com/falkolab/Ti.SvgView/commit/56a04d74783f78690393312b77429a92fb071b0c

**Comment by focussing** [comment]
@falkolab thanks, I already saw that and tried to contact you via Twitter :)
Isn't this already waiting for @GeraudBourdin to merge that with the main stream?
And I also need an iOS solution... My app supports both Android and iOS

**Comment by falkolab** [comment]
Sorry, I have not so mach expirience in iOS. Do you try https://github.com/falkolab/Ti.SvgView/releases ? I do not remember is it work for iOS as is.

**Comment by focussing** [comment]
Yep I checked that.
The release you made to enable blob/url etc as source for SVG file, is there a distribution also?


Best regards, 
Mit freundlichen Grüßen,
Met vriendelijke groet,
Raymond Verbruggen

FOCUSSING B.V.
Mispel 8
7621 WB Borne
Netherlands
+31 6 250 500 69 
mail@focussing.nl <mailto:mail@focussing.nl> 
www.focussing.nl <http://www.focussing.nl/>
